conan/tools/cmake/toolchain/blocks.py

```diff
@@ -559,6 +559,20 @@ def context(self):
         }
 
 
+class PkgConfigBlock(Block):
+    template = textwrap.dedent("""
+        {% if pkg_config %}
+        set(PKG_CONFIG_EXECUTABLE {{ pkg_config }} CACHE FILEPATH "pkg-config executable")
+        {% endif %}
+        """)
+
+    def context(self):
+        pkg_config = self._conanfile.conf.get("tools.gnu:pkg_config", default=False, check_type=str)
```

Comment:
```suggestion
        pkg_config = self._conanfile.conf.get("tools.gnu:pkg_config", check_type=str)
```

Default is not ``False``, but ``None`` (not defined). It only makes sense to default to False if type=bool